src/doom/g_game.c

```diff
@@ -242,6 +242,9 @@ int		bodyqueslot;
  
 int             vanilla_savegame_limit = 1;
 int             vanilla_demo_limit = 1;
+
+// [crispy] store last cmd to track joins
+static ticcmd_t* last_cmd = NULL;
```

Comment:
Calling this "last" instead of "join" because the code that calls `G_CheckDemoStatus` doesn't always know if there is a join happening or not - it might just be the end of demo playback.